luigi/scheduler.py

```diff
@@ -713,7 +458,15 @@ def __init__(self, config=None, resources=None, task_history_impl=None, **kwargs
         :param task_history_impl: ignore config and use this object as the task history
         """
         self._config = config or scheduler(**kwargs)
-        self._state = SimpleTaskState(self._config.state_path)
+
+        from luigi import scheduler_state  # import here since it needs `Worker` from this file
+
+        if self._config.use_sql_state:
+            logger.warn("Using SQL store!")
```

Comment:
is this really a warning?

Reply:
Ah, yeah -- should change this to info. I still think it's a good idea to print it out on startup